backend/judging-algorithm/README.md

```diff
@@ -29,6 +29,8 @@ This outputs our results in to JSON files `hackerResults.json` and `judgeResults
 
 --
 
+NOTE: With the addition of the judging-algorithm backend implementation, you must start up the backend server before running the above script. This can be done by running `ts-node .\app.ts` in the backend folder. Using `yarn dev` will cause `ECONNRESET` error from axios.
```

Comment:
to make section 3 concise, you could just say
terminal 1: cd to backend => run use node 16 => run the ts-node command
terminal 2: open new terminal => run npx ts-node command

Reply:
oh also npm i typescript shouldn't be an issue since it is in package.json